projects/js-packages/components/components/icons/index.tsx

```diff
@@ -241,28 +244,43 @@ const iconsMap = {
 	star: StarIcon,
 	videopress: VideopressIcon,
 	jetpack: JetpackIcon,
+};
+
+const socialIcons = {
 	facebook: FacebookIcon,
 	twitter: TwitterIcon,
 	linkedin: LinkedinIcon,
 	tumblr: TumblrIcon,
 	google: GoogleIcon,
 };
 
+const iconsMap = {
+	...jetpackIcons,
+	...socialIcons,
+};
+
+export type IconsMap = typeof iconsMap;
+
+export type IconSlug = keyof IconsMap;
```

Comment:
Why do we need to export them?

Reply:
I need those in TS conversion of `ProductOffer` component for better/strict typings.